includes/class-wcpdf-endpoint.php

```diff
@@ -129,6 +133,9 @@ public function get_document_link( $order, $document_type, $additional_vars = ar
 		// handle additional query vars
 		$additional_vars = apply_filters( 'wpo_wcpdf_document_link_additional_vars', $additional_vars, $order, $document_type );
 		if ( ! empty( $additional_vars ) && is_array( $additional_vars ) ) {
+			if ( isset( $additional_vars['output'] ) && $this->pretty_links_enabled() ) {
```

Comment:
Since the `add_query_arg()` will update existing queries, I think there is no harm in removing this if-statement.

Reply:
It seems there was some harm, yeah?

Reply:
No, but was adding the `?output=ubl` in front of the pretty URL, which already includes `/ubl`.